static/test/javascripts/conf/adblock.js

```diff
@@ -0,0 +1,9 @@
+module.exports = function (config) {
```

Comment:
We are, but the common tests take ~30s to run while the adblock ones are instant
I am happy to remove these files though and keep them only locally